sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/EliminateSortsSuite.scala

```diff
@@ -284,6 +284,15 @@ class EliminateSortsSuite extends PlanTest {
     comparePlans(optimized, correctAnswer)
   }
 
+  test("SPARK-32318 should not remove orderBy in distribute statement") {
```

Comment:
super nit: in most cases, add `:` in the prefix, `SPARK-32318:`?

Reply:
Oh.. Right. I missed.

Reply:
Yea, I know you just forgot it